src/style.css

```diff
@@ -1,3 +1,84 @@
+
 body {
+margin: 0;
+padding: 0;
+background-color: #fff;
+border: 0;
+}
+
+header {
+display: flex;
+justify-content: space-between;
+align-items: center;
+padding: 0 50px;
+}
+
+.logo {
+display: block;
+line-height: 0;
+}
+
+.header {
+box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+background: #FFF;
+
+}
+
+.nav-link {
+text-decoration: none;
+display: flex;
+align-items: center;
+color: black;
+position: relative;
+text-transform: uppercase;
+padding: 23px 0;
+  }
+
+.nav-list {
   margin: 0;
+display: flex;
+list-style-type: none;
+font-family: Roboto, sans-serif;
```

Comment:
Add fallback font

Reply:
you shouldn't use font-family here. It's enough to set this rule to body or html